src/main/java/com/soit/animal_encyclopedia/AnimalController.java

```diff
@@ -0,0 +1,60 @@
+package com.soit.animal_encyclopedia;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import com.soit.animal_encyclopedia.entity.Animal;
+
+
+@Controller
+@RequestMapping("/Animals")
```

Comment:
Changed the request mapper here so keep it in mind when you guys are accessing this page or creating redirects to other pages